.travis.yml

```diff
@@ -2,7 +2,19 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - master
+
 env:
   global:
   - ENCRYPTION_LABEL: 104fbe69e8fa
   - COMMIT_AUTHOR_EMAIL: travis-ci@w3.org
+  - URL="https://w3c.github.io/permissions/"
+  - DECISION="https://lists.w3.org/Archives/Public/public-webappsec/2015Mar/0170.html"
+  # Exports $TOKEN - the W3C Token needed by Echidna to auto-publish this spec
+  # See: https://github.com/w3c/echidna/wiki/How-to-use-Echidna-with-ReSpec-and-GitHub#the-token
+  - secure: SneznWMBm52ct+mFI6NAVrB8k/BjO/H/hjR1eB3tjPwF0vzpL5fnbD1mmlNmVJMTWMsSUfI917JotA+7lfwoWJuHwlq2WfzQ0ybfvtuqjo7LS22T5X8IFYmCyVNR5asbJha/LNhqUyGDsZlPBgbKK0hqTd8eWjjQTztnlSrorGM=
+
+after_success:
+  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"'
```

Comment:
We could use the condition from https://github.com/w3c/permissions/blob/master/deploy.sh#L8 (`[ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "master" ]`) to allow Travis to run on all branches, but only ping Echidna for master commits.

LGTM either way.